Update zookeeper.md (#9805)
apache/druid: docs/dependencies/zookeeper.md

```

This znode will contain a payload that indicates to the Historical process what it should do with the given segment. When the Historical process is done with the work, it will delete the znode in order to signify to the Coordinator that it is complete.

## Common zookeeper related config

### Znode size limit

> When dealing with large data volumn and a lot of segments, errors can be caused due to Znode size limit exceeded.

Reviewer: ```suggestion
When dealing with large data volume and a lot of segments, errors can be caused due to Znode size limit exceeded.
```

There are at least 3 places the znode size can be configured:
- Druid nodes runtime property:

> `druid.indexer.runner.maxZnodeBytes` works as druid internal znode limit on middleManagers and overlods.

Reviewer: ```suggestion
`druid.indexer.runner.maxZnodeBytes` works as Druid internal znode limit on MiddleManagers, Indexers, and Overlords.
```

- Zookeeper client side config:

> `jute.maxbuffer` Java system property works as a zookeeper client side limit on druid nodes.

Reviewer: ```suggestion
`jute.maxbuffer` Java system property that works as a ZooKeeper client side limit on Druid nodes.
```

- Zookeeper server side config:

> `jute.maxbuffer` Java system property works as a zookeeper server side limit on your zookeeper cluster.

Reviewer: ```suggestion
`jute.maxbuffer` Java system property works as a ZooKeeper server side limit on your ZooKeeper cluster.
```
